WooCommerce/Classes/Extensions/BlazeCampaignListItem+Customizations.swift

```diff
@@ -53,12 +53,12 @@ extension BlazeCampaignListItem.Status {
     }
 
     private enum Localization {
-        static let pending = NSLocalizedString("Pending", comment: "Status name of an active Blaze campaign")
+        static let inModeration = NSLocalizedString("In Moderation", comment: "Status name of a pending Blaze campaign")
```

Comment:
Nit: Please follow the new rule of separate keys and values for localized strings p91TBi-aJl-p2. I believe it's worth it to update other strings in the enum as well.

Reply:
Done here 5a27ac1f140817a0cc4a94c4749852f87609344f I updated also the other `NSLocalizedString` in this file.